File: SistemaRural_PE/src/modelo/Paciente.java
package modelo;

//2. Paciente.java
import java.util.List;

public class Paciente extends Persona {
 private String numeroHistoriaClinica;
 private String tipoSeguro;
 private HistorialMedico historialMedico = new HistorialMedico();
 //Getters and Setters
 public String getNumeroHistoriaClinica() {
	return numeroHistoriaClinica;
}


 public void setNumeroHistoriaClinica(String numeroHistoriaClinica) {
	this.numeroHistoriaClinica = numeroHistoriaClinica;
 }


 public String getTipoSeguro() {
	return tipoSeguro;
 }


 public void setTipoSeguro(String tipoSeguro) {
	this.tipoSeguro = tipoSeguro;
 }


 public HistorialMedico getHistorialMedico() {
	return historialMedico;
 }


 public List<CitaMedica> getCitasMedicas() {
	return historialMedico.getCitas();
 }

 
 
 

 //Clase solicitarCita con parámetro objeto de CitaMedica
 //Agregar objeto de clase a la Lista citasMedicas
 public void solicitarCita(CitaMedica cita) {
     historialMedico.agregarCita(cita);
 }

 
 //Agregar un método público consultarHistorial
 public List<CitaMedica> consultarHistorial() {
     return historialMedico.consultarAtenciones();
 }
}

File: Clases de Python en el Sistema Rural/Paciente.py
# paciente.py
from Persona import Persona
from Historial_Medico import HistorialMedico

class Paciente(Persona):
    def __init__(self):
        super().__init__()
        self.numero_historia_clinica = ""
        self.tipo_seguro = ""
        self.historial_medico = HistorialMedico()

    def get_citas_medicas(self):
        return self.historial_medico.get_citas()

    def solicitar_cita(self, cita):
        self.historial_medico.agregar_cita(cita)

    def consultar_historial(self):
        return self.historial_medico.consultar_atenciones()